### PR TITLE
Spelling fixed

### PR DESCRIPTION
**Pull request by PickleEric**

### CODE REVIEW

**Comment by claraj** [comment]
@svenJonis138 can you review?

**Comment by svenJonis138** [comment]
Just got finished with mine, glad you got it working!
SJ

File: convert_state.py [hunk 1]
```diff
@@ -1,60 +1,58 @@
...
+import json  # import json for json processing
 
 
-
-import json  # import json for json processing
 def main():
+
+    """ Main function creates dictionary of states and runs menu """
+
     with open('data.json','r') as file:
```
Comment (claraj) [inline convert_state.py]: Space after comma in arguments

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,58 @@
...
+        state_key_and_abbrev[statevalue] = statekey
 
-
-    #loop
     while True:
         print('1. Convert state to abbreviation')
-        print("2. Convery abreviasion to state")
-        print(' 3. quit')
-        choice =   input("Enter choice : ")
+        print("2. Convert abbreviation to state")
```
Comment (claraj) [inline convert_state.py]: Use single quotes

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,58 @@
...
+        print('3. Quit')
+        choice = input("Enter choice:")
 
         if choice=="1":
```
Comment (claraj) [inline convert_state.py]: Single quotes for all strings, put a space before and after the == operator `if choice == '1':`

File: convert_state.py [hunk 4]
```diff
@@ -1,60 +1,58 @@
...
 
+def convert_state_to_abbreviation(dictionary ):
 
+    """ Take state name and convert to abbreviation"""
 
+    user_input = input("Enter state name:").capitalize()
+    result = dictionary.get(user_input)
 
+    if result is None:
+        print('State Not found')
```
Comment (claraj) [inline convert_state.py:42]: Be consistent with capitalization

File: convert_state.py [hunk 5]
```diff
@@ -1,60 +1,58 @@
...
+    else:
+        print ( "The abbreviation for " + user_input + " is " + result + ".")
```
Comment (claraj) [inline convert_state.py]: recommend f-string

File: convert_state.py [hunk 6]
```diff
@@ -1,60 +1,58 @@
...
-    main()
\ No newline at end of file
+    """ Take abbreviation to state name"""
+
+    user_input = input("Enter abbrviation name:").upper()
+    result = dictionary.get(user_input)
+    if result is None:
+        print('Abreviation not found.')
+    else:
+        print("The state with the abbreviation." + user_input + " is " + result)
```
Comment (claraj) [inline convert_state.py]: recommend f-string

**Review by claraj — CHANGES_REQUESTED** [review]
Looking much better! Few things to take a look at.

**Pull request closed without merging**